src/metrics-server/templates/metrics-server-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "metrics-server.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "metrics-server.labels" . | indent 4 }}
+    kubernetes.io/name: "Metrics-server"
```

Comment:
Out of curiosity, what does this label do for us?

Reply:
I don't know. As I said in the internal PR, I did not write this chart.